web/src/app.svelte

```diff
@@ -54,9 +54,7 @@
   {#if !$captureMode}
     <div class="flex justify-between px-2 sm:px-4 py-2 items-center">
       <div>
-        <h1 class="text-2xl sm:text-3xl text-sky-500 dark:text-sky-400">
-          {'<' + import.meta.env.VITE_SITE_NAME + ' />'}
-        </h1>
+          <img src="Logo_glitch.svg" alt="Logo" width="40%">
```

Comment:
For accessibility and SEO, recommend setting the `alt` text to the website title.
 
```suggestion
          <img src="Logo_glitch.svg" alt="thai-tech-calendar" width="40%">
```